index.html

```diff
@@ -81,25 +82,101 @@ <h1 class="main-head">
               />
             </div>
           </div>
-          <div class="d-flex align-items-center rectangle-with-icons">
-            <ul>
-              <li>
-                <div class="circle"></div>
-                <span>90+</span>
-                <p>Users</p>
+          <div class="box-flex">
+            <div class="box-shadow" aria-hidden="true"></div>
```

Comment:
You don't need aria-hidden for this element.

Reply:
Ok, understood.